packages/cli/src/lib/project/project.ts

```diff
@@ -106,9 +106,13 @@ export class Project {
   }
 
   public writeResourcesManifest(orgId: string) {
-    const manifestJson =
-      readJsonSync(this.resourceManifestPath, {throws: false}) ?? {};
-    writeJsonSync(this.resourceManifestPath, {...manifestJson, orgId});
+    try {
+      const manifestJson =
+        readJsonSync(this.resourceManifestPath, {throws: false}) ?? {};
+      writeJsonSync(this.resourceManifestPath, {...manifestJson, orgId});
```

Comment:
When would this fail?

Reply:
It could fail if for some odd reason the file is read-only or if the file does not exist and the directory is read-only.
TL;DR: odd chmod could cause EACCESS issue, theoretically